rakudup

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env bash
 # https://github.com/stmuk/rakudup
 
-RAKUDO_ROOT="${HOME}/.rakudup"
+RAKUDO_ROOT=$HOME/.rakudup
```

Comment:
Why did you remove "" here?

Reply:
No need really, just for consistency with the other line below.